src/resources/Correction/BenefitCompanyStatementResource.ts

```diff
@@ -13,7 +13,13 @@ export const BenefitCompanyStatementResource = {
       CorrectionTypes.CORRECT_NEW_NR,
       CorrectionTypes.CORRECT_NAME_TO_NUMBER,
       CorrectionTypes.CORRECT_NAME
-    ]
+    ],
+    orgPersonInfo: {
+      orgPersonLabel: 'People and Roles',
+      subtitle: 'You must have a minimum of two partners. You can add or remove partners (individual person or ' +
+        'business) as well as change the mailing and delivery addresses and email address of individual people and ' +
+        'business partners that were added manually during registration.'
```

Comment:
Where did this come from?

Reply:
Probably a copy past mistake. Didn't notice it because it doesn't render for non firms. Will remove.

Reply:
> Nice looking code! I realize some of it is in flux atm.
> 
> Do you need a FF, or no because there are no firms in the system atm?
> 
> I assume you've fully tested these changes w.r.t. other entity types. Please also call out these changes so Riyaz learns about them.

Yea, the dev work has sort of a required a Correction/Alteration/ChangeOfReg filing to be open at all times, at least thats how i do it. Especially since we were pulling some bits out and into resource files. 

We currently have the flag to protect the filing as a whole, which should be enough for now! Only changes visible in our Correction filing is aesthetic stuff, layout improvements :)